airbyte-integrations/connectors/source-primetric/source_primetric/schemas/assignments.json

```diff
@@ -339,6 +381,242 @@
+            "number"
+          ]
+        }
+      }
+    },
+    "financial_total_scheduled_cost" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
+        "amount" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        },
+        "currency" : {
+          "type" : [
+            "null",
+            "string"
+          ]
+        },
+        "exchange_rate" : {
+          "type" : [
+            "null",
+            "integer"
+          ]
+        }
+      }
+    },
+    "financial_total_tracked_cost" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
+        "amount" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        },
+        "currency" : {
+          "type" : [
+            "null",
+            "string"
+          ]
+        },
+        "exchange_rate" : {
+          "type" : [
+            "null",
+            "integer"
+          ]
+        }
+      }
+    },
+    "financial_total_tracked_income" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
+        "amount" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        },
+        "currency" : {
+          "type" : [
+            "null",
+            "string"
+          ]
+        },
+        "exchange_rate" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        }
+      }
+    },
+    "financial_settled_income" : {
+      "type" : [
+        "null",
+        "object"
+      ]
+    },
+    "financial_settled_cost" : {
+      "type" : [
+        "null",
+        "object"
+      ]
+    },
+    "financial_total_work_for_cost" : {
+      "type" : [
+        "null",
+        "integer"
+      ]
+    },
+    "financial_contractor_hour_cost" : {
+      "type" : [
+        "null",
+        "object"
+      ]
+    },
+    "financial_hour_rate" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
+        "amount" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        },
+        "currency" : {
+          "type" : [
+            "null",
+            "string"
+          ]
+        },
+        "exchange_rate" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        }
+      }
+    },
+    "financial_employee_default_hour_cost" : {
+      "type" : [
+        "null",
+        "object"
+      ],
+      "properties" : {
+        "amount" : {
+          "type" : [
+            "null",
+            "number"
+          ]
+        },
+        "currency" : {
+          "type" : [
+            "null",
+            "string"
+          ]
+        },
+        "exchange_rate" : {
+          "type" : [
+            "null",
+            "integer"
+          ]
+        }
+      }
+    },
+    "financial_use_default_hour_cost" : {
```

Comment:
Have you seen this type in the api response ? The name suggests a boolean but I can't check it in the api doc 🤔

Reply:
In my case we do not use the financial fields, I checked, indeed it should be boolean, my mistake :smile_cat: